package.json

```diff
@@ -31,6 +31,8 @@
   },
   "homepage": "https://github.com/neomad-team/neomad.org#readme",
   "dependencies": {
+    "preact": "^8.1.0",
+    "preact-compat": "^3.16.0",
     "react": "^15.5.4",
```

Comment:
Do you still need react, react-dom and others as dependencies if you use Preact?
I would thing that [importing it from Preact](https://github.com/neomad-team/neomad.org/pull/88/commits/7c9bae0692cda46fda500a84a0d957d40c174e88#diff-0370a95ba252f31f1b9df66f778aad84R1) and [defining it as alias](https://github.com/neomad-team/neomad.org/pull/88/commits/7c9bae0692cda46fda500a84a0d957d40c174e88#diff-b83f3305d04bb1c8d674be30b5f9e1efR24) would do the job.